src/index.html

```diff
@@ -17,6 +17,12 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+
+    <section class="stopwatch">
```

Comment:
The `speed-up` modifier class is missing from the stopwatch section. According to the task requirements, this modifier should be implemented but not included by default. Ensure that the modifier can be added or removed as needed for testing purposes.